### DISCUSSION TITLE
Field names containing slashes

### DISCUSSION THREAD

**Discussion by mgoodness** [cuelang/cue#501] [General] [answered]
In this command definition, how do I output the value of the `app.kubernetes.io/component` label of each object?

```
command: ls: {
  task: print: cli.Print & {
    text: tabwriter.Write([
      for x in objects {
        "\(x.kind)  \t\(x.metadata.labels[app.kubernetes.io/component])  \t\(x.metadata.name)"
      },
    ])
  }
}
```

**Comment by mpvl** [comment, marked as answer]
You should use quotes here, so `"app.kubernetes.io/component"`

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/501.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
